processor/transform/index.ts

```diff
@@ -8,7 +8,15 @@ import injectComponents from './inject-components';
 import readmeComponentsTransformer from './readme-components';
 import readmeToMdx from './readme-to-mdx';
 import variablesTransformer from './variables';
+import tablesToJsx from './tables-to-jsx';
 
-export { readmeComponentsTransformer, readmeToMdx, injectComponents, variablesTransformer };
+export {
+  divTransformer,
+  readmeComponentsTransformer,
+  readmeToMdx,
+  injectComponents,
+  variablesTransformer,
+  tablesToJsx,
+};
 
-export default [calloutTransformer, codeTabsTransfromer, divTransformer, embedTransformer, imageTransformer, gemojiTransformer];
+export default [calloutTransformer, codeTabsTransfromer, embedTransformer, imageTransformer, gemojiTransformer];
```

Comment:
I removed `divTransformer` from the default export since those get used in the `compile` step. But I think we only need in `mdx` for the migration?